README.md

```diff
@@ -40,3 +40,12 @@ lastly, you would need to boot ipxe, a simple way to test on a vm, while ipxe is
 dhcp
 chain <your base url>/ipxe
 ```
+
+## Use as script
+
+If you cannot run the container, you can run the script directly:
```

Comment:
maybe provide examples of packages / binaries are required in that case?

Reply:
added